.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Oops, I guess that just slipped in by accident.

Reply:
Damn. I've really gotta commit that to the .gitignore soon...